+precice/@SolverInterface/private/preciceGateway.cpp

```diff
@@ -60,7 +60,13 @@ enum class FunctionID {
     readBlockVectorData = 68,
     readVectorData = 69,
     readBlockScalarData = 70,
-    readScalarData = 71
+    readScalarData = 71,
```

Comment:
What is exactly happening here? I guess it may be some editing thing, but I am not sure why it pops up.

Reply:
Adding new function to following lines requires a comma `,` at the end of the line

Reply:
Ah, indeed. I overlooked this.